### ISSUE TITLE
Add pre-commit hook

### ISSUE THREAD

**Issue by getzze** [Diaoul/subliminal#1075]
Only after `ruff` is happy :)
https://github.com/pre-commit/pre-commit

**Comment by getzze** [comment]
I think it would be great to have https://pre-commit.ci/
That an automatic fixer for PR. I think only the owner of the repo can subscribe (it's free) @Diaoul

**Comment by getzze** [comment]
this is not a priority anymore
